legend-engine-core/legend-engine-core-base/legend-engine-core-language-pure/legend-engine-language-pure-compiler/pom.xml

```diff
@@ -260,6 +260,10 @@
             <artifactId>hamcrest-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
This feels like the wrong thing to do: we shouldn't be adding a dependency from the core compiler to an xt module.